### ISSUE TITLE
Process SWMM report, stop SWMM if inp has errors

### ISSUE THREAD

**Issue by tproduit** [QGEP/QGEP#613]
Currently, when the inp file generated contains errors, SWMM fails and make QGIS crash.

1. SWMM should be stopped if the inp file contains errors
2. The status report should be parsed in order to generate a temporary layer with the locations of the errors and an hint on how to solve the error.

Errors that we have encountered until now
```
`Undefined Node (CHamtKnv00002706) referenced at line 20997:
[DWF]
CHamtKnv00002706	FLOW	0.0204074074074074	dailyPatternDWF
```
The catchment area is linked to a secondary node
![image](https://user-images.githubusercontent.com/8256037/99684534-dc4b4400-2a81-11eb-9907-234665eb01c0.png)

```
Undefined Node (CHamtKnv00013337) referenced at line 22051:
[CONDUIT]
CHamtKnv00000509	CHamtKnv00002509	CHamtKnv00013337
```
A side reach connect "blindly" to a main reach. According to VSA, the main reach should be cut at their crossing.
![image](https://user-images.githubusercontent.com/8256037/99684521-d6edf980-2a81-11eb-8ae2-13d07a60ff55.png)

**Comment by tproduit** [comment]
@sjib does this issue match our discussion?

**Comment by sjib** [comment]
Yes, I updated the description to be a little bit more precise. If the side reach would be a secondary wastewater structure the topology would be ok like this.

**Comment by tproduit** [comment]
```
Too few items at line 104:
[SUBCATCHMENT]
ch19ar68CA008855_ww_current	raingage@ch19ar68CA008855	CHamtKnv00001992	0.0276		19.4322613668305	0.5	0
```
The subcatchement discharge coefficients are not filled

**Comment by tproduit** [comment]
```
  ERROR 211: invalid number dailyPatternDWF at line 4288 of [DWF] section:
  00000000WN006032 FLOW             dailyPatternDWF
```

 `qgep_od.catchment_area.population_density_current` or `qgep_od.catchment_area.population_density_planned` are not filled
